web/js/components/context-menu/context-menu.js

```diff
@@ -68,13 +74,38 @@ function RightClickMenu(props) {
     };
   };
 
+  let longPressTimer = null;
+
+  function setTimer(incomingEvent, incomingOlMap) {
+    longPressTimer = setTimeout(() => {
+      setIsHoldPress(true);
```

Comment:
It seems odd that this gets set to true and never will be reset to false at any point.  There is an issue I am seeing that might be related.  Sometimes a click and hold mouse interaction to drag the map causes the context menu to appear.  And also, sometimes right-clicking after left click causes the context menu to appear and then immediately disappear.

Reply:
Here is a demonstration: https://drive.google.com/file/d/11mbMyl61-Ek3wj1Ug05n97IcQ0l-TVH8/view?usp=sharing